README.md

```diff
@@ -60,229 +61,184 @@ After you're finished please remove all the comments and instructions!
+- **form validation**
+- **store form data in local storage**
+- **Added the single Page behaviors**
+- **Implementation of ES6 and modules**
+- **Webpack plugins**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
+- Comming Soon...
 
-- [Live Demo Link](https://google.com)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- WalkThrough Video -->
+
+## 🦻 WalkThrough Video <a name="#walkthrough-demo"></a>
+
+-  Comming Soon...
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
 To get a local copy up and running, follow these steps.
 
 ### Prerequisites
 
 In order to run this project you need:
+- Visual Studio Code.
+- Node JS.
+- Git bash.
+- GitHub Account.
 
 <!--
 Example command:
-
 ```sh
  gem install rails
 ```
  -->
-
 ### Setup
 
 Clone this repository to your desired folder:
+Use git clone command or download ZIP folder
 
 <!--
 Example commands:
 
 ```sh
   cd my-folder
-  git clone git@github.com:myaccount/my-project.git
+  git clone git@github.com:KUBAHO3/Hello-Microverse.git
 ```
 --->
-
 ### Install
-
 Install this project with:
-
-<!--
+npm
 Example command:
-
 ```sh
   cd my-project
-  gem install
+  npm init -y
 ```
---->
-
 ### Usage
-
 To run the project, execute the following command:
-
-<!--
+npm start or live server
 Example command:
-
 ```sh
-  rails server
+  npm run build
 ```
---->
-
 ### Run tests
-
 To run tests, run the following command:
-
-<!--
+npm test
 Example command:
-
 ```sh
-  bin/rails test test/models/article_test.rb
+  npx stylelint "**/*.{css,scss}"
+  npx eslint .
 ```
---->
-
 ### Deployment
-
 You can deploy this project using:
-
-<!--
+GitHub Pages
 Example:
-
 ```sh
-
+git@github.com:KUBAHO3/Sharing-World.git
 ```
- -->
-
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- AUTHORS -->
-
-## 👥 Authors <a name="authors"></a>
+## :busts_in_silhouette: Authors <a name="authors"></a>
+:bust_in_silhouette: **KUBAHO LINNE Heaven**
+- GitHub: [@KUBAHO3](https://github.com/KUBAHO3)
+- Twitter: [@KubahoH](https://twitter.com/KubahoH)
+- LinkedIn: [Kubaho Linne Heaven](https://www.linkedin.com/in/kubaho-linne-heaven-78ab37208/)
 
-> Mention all of the collaborators of this project.
-
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+:bust_in_silhouette: **Yusuf Sholotan**
+- GitHub: [@yin-ka](https://github.com/yin-ka)
```

Comment:
- [ OPTIONAL ] This is an individual project but you have 2 authors here. Kindly fix this. 🤝  
![authos](https://user-images.githubusercontent.com/99656687/231116705-e4275e2d-81a8-4cd8-b503-31d58133eac8.png)

Reply:
updated to meet requirement